package.json

```diff
@@ -85,7 +85,7 @@
         "clean": "echo \"Remove $(git rev-parse --show-toplevel)/node_modules\" && lerna clean && rm -rf \"$(git rev-parse --show-toplevel)/node_modules\"",
         "deploy": "f () { export NODE_ENV=staging && npm run build:all && lerna exec --scope @deriv/core -- npm run deploy $@ && export NODE_ENV= ;}; f",
         "deploy:clean": "f () { npm run build:travis && npm run build:local && lerna exec --scope @deriv/core -- npm run deploy:clean $@ ;}; f",
-        "deploy:folder": "f () { export NODE_ENV=staging && npm run build:gh-pages $1 && lerna exec --scope @deriv/core -- npm run deploy:folder $@ && export NODE_ENV= ;}; f",
+        "deploy:folder": "f () { export NODE_ENV=staging && npm run build:all && lerna exec --scope @deriv/core -- npm run deploy:folder $@ && export NODE_ENV= ;}; f",
```

Comment:
@maryia-deriv We have to pass in the $1 to get the asset path working correctly if we publish the build files into a folder.

Reply:
@balakrishna-deriv unfortunately, it doesn't publish at all with `npm run build:gh-pages $1`, it's failing with the following error:
<img width="1326" alt="Screenshot 2023-04-13 at 10 43 39 AM" src="https://user-images.githubusercontent.com/103177211/231689508-41c1693b-9290-4aa2-abba-ff638f26cc7f.png">
Whereas `npm run build:all` works great, just like it does in `npm run deploy` script.🙂

Reply:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/56330681/231953126-d96d7f3c-ad9a-446f-8f4a-1ea1786411a3.png">
@maryia-deriv Can you try this change instead? The build works for me with this change.

Reply:
working👍🏻 thank you!